components/gitpod-db/leeway.Dockerfile

```diff
@@ -2,19 +2,20 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM node:12.18.3-slim as builder
+FROM node:12.18.3 as builder
 COPY components-gitpod-db--migrations /installer/
 
 WORKDIR /app
 RUN /installer/install.sh
 
 
-FROM node:12.18.3-slim
+FROM node:12.18.3
```

Comment:
instead of making the db-migrations image larger you could also resort to a multi-stage build that downloads the `cloud_sql_proxy`.